doc/user/actions.rst

```diff
@@ -92,6 +92,9 @@ when you click anywhere on the bar (where there isn't another click action):
 This will then trigger the ``toggle`` action on the ``mydate`` module when you
 click anywhere on the bar.
 
+/!\\ ``click-left`` and related trigger configurations are only possible in the bar section,
```

Comment:
I'm not sure what this thing at the beginning is supposed to do, but it doesn't render as anything special on readthedocs: https://polybar--2881.org.readthedocs.build/en/2881/user/actions.html#triggering-actions

Reply:
It's not supposed to render anything else than a warning sign : /!\

Reply:
If I may, this is not following the rst habbits. As I was looking for the same kind of information, I agree to add this paragraph in the documentation, but instead of prefixing it with `/!\`, you should put it under as a note admonition block:

```
.. note::
  ``click-left`` and related trigger configurations are only possible in the bar 
  section, as well as in the following modules: ``text``, ``script``, ``ipc``.
```